internal/probeservices/probeservices.go

```diff
@@ -32,8 +32,8 @@ import (
 )
 
 var (
-	// ErrUnsupportedEndpoint indicates that we don't support this endpoint type.
-	ErrUnsupportedEndpoint = errors.New("probe services: unsupported endpoint type")
+	// ErrUnsupportedServiceType indicates that we don't support this service type.
+	ErrUnsupportedServiceType = errors.New("probe services: unsupported service type")
```

Comment:
We generally do not change error strings. However, this error string is for a very uncommon case, because the service type should always be supported and so it's fine to modify it for consistency.